### ISSUE TITLE
[gulliver-modules] fortytwo (Trac #1360)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1352]
`fortytwo` is used throughout the gulliver suite, but is in `gulliver-modules/resources/test`.  Find a better place to put this, so you don't have to mess with `sys.path` in order to import it.

This will also fix the buildbots, so would be a "good thing".

Migrated from https://code.icecube.wisc.edu/ticket/1360
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "`fortytwo` is used throughout the gulliver suite, but is in `gulliver-modules/resources/test`.  Find a better place to put this, so you don't have to mess with `sys.path` in order to import it.\n\nThis will also fix the buildbots, so would be a \"good thing\".", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[gulliver-modules] fortytwo", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-09-21T21:59:33", 
    "milestone": "", 
    "owner": "kkrings", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150921T22:30:10: 
* kkrings changed owner from "olivas" to "kkrings"
* kkrings changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150921T22:30:31: kkrings changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150922T00:05:34: 
* kkrings commented:

> In changeset 137813:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137813"
> Moved fortytwo.py to gulliver-modules/python and updated all scripts that import it; fixes #1352.
> ```


* kkrings changed resolution from "" to "fixed"
* kkrings changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
